iohk-monitoring/src/Cardano/BM/Backend/Log.lhs

```diff
@@ -269,8 +269,8 @@ passN backend katip (LogObject loname lometa loitem) = do
                                      (severity lometa, "", Just . Right $ Object s)
                                 (LogValue name value) ->
                                     if name == ""
-                                    then (severity lometa, pack (showSI value), Nothing)
-                                    else (severity lometa, name <> " = " <> pack (showSI value), Nothing)
+                                    then (severity lometa, pack (showSI value), Just $ Left loitem)
```

Comment:
I can understand that the `LogError` is a `Left`, but why is the `LogValue` a `Left` as well?
Shouldn't this be a `Right`?

Reply:
LogValue is also a LOContent, like LogError.